test/functional_test.js

```diff
@@ -1392,10 +1392,7 @@ function runOnPrefix(apiPrefix) {
             conf.set('hekaMetrics', hekaMetrics);
             sandbox.stub(hekaLogger, "info", function(op, log) {
               log.op = op;
-              try {
-                _logs.push(log);
-              } catch (e) {
```

Comment:
This was done by @tarekziade in https://github.com/mozilla-services/loop-server/commit/eef15f3217e18d58348c80e63ffb738a0d9acbc9 I don't quite understand why this is from the commit message. I would be tempted to just remove it.

@tarekziade any insights?